packages/expo-splash-screen/src/SplashScreen.types.ts

```diff
@@ -0,0 +1,20 @@
+export type SplashScreenOptions = {
+  /**
+   * The duration of the fade out animation in milliseconds.
+   * @default 400
+   */
+  duration?: number;
+  /**
+   * Whether to hide the splash screen with a fade out animation.
+   * @ios
+   * @default false
+   */
+  fade?: boolean;
+};
+
+export type SplashScreenNativeModule = {
+  setOptions: (options: SplashScreenOptions) => void;
```

Comment:
do you always require users to call hide? why not set the options as a hide() parameter over having a separate method?